package.json

```diff
@@ -35,32 +49,24 @@
   },
   "scripts": {
     "bootstrap": "lerna bootstrap --no-ci",
-    "build": "lerna run build  --stream",
-    "build:alexa-types": "yarn --cwd packages/alexa-types build",
-    "build:all": "yarn build",
-    "build:api-sdk": "yarn --cwd packages/api-sdk build",
-    "build:base-types": "yarn --cwd packages/base-types build",
-    "build:chat-types": "yarn --cwd packages/chat-types build",
-    "build:common": "yarn --cwd packages/common build",
-    "build:general-types": "yarn --cwd packages/general-types build",
-    "build:google-dfes-types": "yarn --cwd packages/google-dfes-types build",
-    "build:google-types": "yarn --cwd packages/google-types build",
-    "build:metrics": "yarn --cwd packages/metrics build",
-    "build:voice-types": "yarn --cwd packages/voice-types build",
+    "build": "yarn build:cjs && yarn build:esm",
+    "build:cjs": "ttsc --build",
+    "build:esm": "ttsc --build tsconfig.esm.json",
     "clean": "yarn clean:artifacts && yarn clean:root",
     "clean:artifacts": "lerna run clean --parallel",
     "clean:root": "rimraf node_modules",
     "commit": "cz",
-    "lint": "lerna run lint  --parallel",
-    "lint:fix": "lerna run lint:fix  --parallel",
-    "lint:quiet": "lerna run lint:quiet  --parallel",
-    "lint:report": "lerna run lint:report  --parallel",
-    "postinstall": "yarn bootstrap",
+    "lint": "lerna run lint --parallel",
+    "lint:fix": "lerna run lint:fix --parallel",
+    "lint:quiet": "lerna run lint:quiet --parallel",
+    "lint:report": "lerna run lint:report --parallel",
+    "postinstall": "[[ -e node_modules/_bootstrap ]] && lerna bootstrap && rm node_modules/_bootstrap || true",
```

Comment:
now `lerna bootstrap` should only run on the initial install
if we add future packages we'll need to run `yarn bootstrap` to link them
however I've found having the `bootstrap` always run was occasionally causing issues with the install process